pkg/reconciler/instances/istio/reset/data/gatherer_test.go

```diff
@@ -103,6 +106,7 @@ func fixPodWith(name, namespace, image, phase string) *v1.Pod {
 			OwnerReferences: []metav1.OwnerReference{
 				{Kind: "ReplicaSet"},
 			},
+			Annotations: map[string]string{"sidecar.istio.io/status":"true"},
```

Comment:
What about creating some with pods this annotation set to `true` and some to `false`? Did you think about such unit test cases?

Reply:
Already adjusted. Does not matter what is the value of this annotation. If sidecar injection is enabled than this annotation exists